docs/release-notes/v12.2.2.md

```diff
@@ -0,0 +1,146 @@
+
+![Choose Crud Index](./img/v12.2.2/crud-sorting-index.png)
+
+:::info
+This field will be displayed only if the `crud-service` version is greatern than `v7.0.0`.
+:::
+
+### Bug Fix
+
+This version addressed the following bug:
+
+* fixed a bug that sent the user back to the Settings section instead of Homepage upon clicking on the Overview area of the project;
+* miactl: a bug causing a malfunction on the deploy command has been fixed;
+* miactl: a bug on opening the browser during login has been fixed.
+
+## Fast Data
+
+### Fast Data Configurator
+
+#### Improvements
+
+##### Default Sorting Index for Single Views
+
+The [default sorting index feature](#default-sorting-index-for-crud-collections) added to CRUD Collections is also available for Single View collections.
+
+##### Better error handling inside Aggregation No Code for Single View key fields
+
+The No Code Aggregation section has been revamped to display only fields that are already mapped or Single View key fields, reducing clutter and improving clarity. This is especially useful for Single Views with lots of fields in their Data Model. When needed, a new button will allow to add unmapped fields inside the section.
+
+### Fast Data Services
+
+#### Real-Time Updater
+
+The new version `v7.5.8` of the _Real-Time Updater_ is available!
+
+##### Improvements
+
+###### Support for events compression
+
+Starting from this version, the service can now produce events using the supported compression algorithms, that are:
+
+- `gzip`
+- `snappy`
+
+This yields an improvement over the storage usage on the Kafka broker, since less space is necessary to store the events
+generated by Fast Data service. To configure such feature, please add the new environment variable `PRODUCER_COMPRESSION`
+with the necessary value.
+
+:::info
+Real-Time updater consumer already supported compression on the consumer component, so that change events sent from your System Of Record systems can be compressed too.
+:::
+
+For more information check out the service [changelog](/runtime_suite/real-time-updater/changelog.md).
+
+#### Single View Trigger Generator
+
+The new version `v3.1.6` of the _Single View Creator_ is available!
+
+##### Improvements
+
+###### Support for events compression
+
+Starting from this version, the service can now consume and produce events using the supported compression algorithms, that are:
+
+- `gzip`
+- `snappy`
+
+This yields an improvement over the storage usage on the Kafka broker, since less space is necessary to store the events
+generated by Fast Data service. To configure such feature on the producer side, please add the `compressionName` property with needed value
+to the Event Store _producer_ configuration. On the contrary, the consumer component is automatically enabled to read events that have been compressed.
```

Comment:
```suggestion
This yields an improvement over the storage usage on the Kafka broker, since less space is necessary to store the events generated by Fast Data service. To configure such feature on the producer side, please add the `compressionName` property with needed value to the Event Store _producer_ configuration. On the contrary, the consumer component is automatically enabled to read events that have been compressed.
```